service_container/autowiring.rst

```diff
@@ -204,24 +205,25 @@ subsystem isn't able to find itself the interface implementation to register:
             xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
             <services>
-                <service id="rot13_transformer" class="Acme\Rot13Transformer" />
+                <service id="AppBundle\Rot13Transformer" />
 
-                <service id="twitter_client" class="Acme\TwitterClient" autowire="true" />
+                <service id="AppBundle\TwitterClient" autowire="true" />
             </services>
         </container>
 
     .. code-block:: php
 
-        use Acme\TwitterClient;
+        use AppBundle\Rot13Transformer;
+        use AppBundle\TwitterClient;
 
         // ...
-        $container->register('rot13_transformer', 'Acme\Rot13Transformer');
-        $container->autowire('twitter_client', TwitterClient::class);
+        $container->register(Rot13Transformer::class);
+        $container->autowire(TwitterClient::class);
 
-The autowiring subsystem detects that the ``rot13_transformer`` service implements
-the ``TransformerInterface`` and injects it automatically. Even when using
-interfaces (and you should), building the service graph and refactoring the project
-is easier than with standard definitions.
+The autowiring subsystem detects that the ``Rot13Transformer::class`` service
+implements the ``TransformerInterface`` and injects it automatically. Even when
```

Comment:
Autowiring using types still works but is deprecated (`TransformerInterface` is detected on `Rot13Transformer`), do you mind if we reword this more deeply this later in another PR? Or maybe I can simply add a note that it is deprecated?